docs/dev/dev_install.rst

```diff
@@ -97,8 +97,8 @@ After the environment file has been created you can create an environment out of
 
 .. code-block:: console
 
-    $ micromamba env create -f environment.yml
-    $ micromamba activate hydromt
+    $ mamba env create -f environment.yml
+    $ mamba activate hydromt
```

Comment:
Should this be hydromt-full if we create it with `make dev`?
In the text above I think it's also better to focus on the flavours only. We can move the more detailed descriptions to a new section further below. 
```suggestion
    $ mamba activate hydromt-full
```

Reply:
good point, I think I forgot to update that one